docs/emulate.rst

```diff
@@ -0,0 +1,62 @@
+.. _emulate:
+
+
+Prepare QEMU
+------------
+
+You will need to prepare QEMU STM32 emulator from:
+`QEMU STM32 v0.1.3 <https://github.com/beckus/qemu_stm32/releases>`_
+
+After download, compile the qemu_stm32 by::
+
+    $ ./configure --disable-werror --enable-debug \
+            --target-list="arm-softmmu" \
+            --extra-cflags=-DDEBUG_CLKTREE \
+            --extra-cflags=-DDEBUG_STM32_RCC \
+            --extra-cflags=-DDEBUG_STM32_UART \
+            --extra-cflags=-DSTM32_UART_NO_BAUD_DELAY \
+            --extra-cflags=-DSTM32_UART_ENABLE_OVERRUN --python=python2
+    $ make -j8
+
+
+Make sure you have export the `arm-softmmu` to `$PATH`::
+
+    $ export PATH=$PATH:~/qemu_stm32-stm32_v0.1.3/arm-softmmu
+
+
+Run Piko/RT on QEMU STM32-p103
+------------------------------
+
+You will need to run these command::
+
+    $ make TARGET=stm32p103
+    $ make TARGET=stm32p103 run
```

Comment:
You say that the ability to check build target run with the correct machine (e.g. target=stm32f429 but run QEMU with `-machine stm32p103`)?

Or to check qemu-system-arm support `-machine stm32p103`, if is this, this should be cover by QEMU its self?